app/src/main/java/eu/kanade/tachiyomi/ui/manga/MangaScreenModel.kt

```diff
@@ -718,6 +721,27 @@ class MangaScreenModel(
                 read = read,
                 chapters = chapters.toTypedArray(),
             )
+
+            if (read) {
+                val maxChapterNumber = chapters.maxOf { it.chapterNumber }
+
+                val formattedChapterNumber = if (maxChapterNumber % 1.0 == 0.0) {
+                    maxChapterNumber.toInt().toString()
+                } else {
+                    maxChapterNumber.toString()
+                }
+
+                val result = snackbarHostState.showSnackbar(
+                    message = "Update trackers to chapter $formattedChapterNumber?",
+                    actionLabel = "Yes",
```

Comment:
...

Reply:
Why bother with i18n if people hate the concept, just leaving this PR here to get opinions

Reply:
Unless you want me to switch it now

Reply:
Yes

Reply:
done now.

Reply:
Should I also prompt if the user marks a chapter as unread?